### ISSUE TITLE
vagrant(v1.2.2) display "Failed to load the "vagrant-berkshelf" plugin" after install

### ISSUE THREAD

**Issue by simoniong** [berkshelf/vagrant-berkshelf#47]
vagrant ( v 1.2.2 ) seems can not install plugin vagrant-berkshelf very well, here's the symptoms when I try to install this plugin in Mac OS X.

``` bash
>>vagrant -v
Vagrant version 1.2.2

>>vagrant plugin install vagrant-berkshelf
Installing the 'vagrant-berkshelf' plugin. This can take a few minutes...
Installed the plugin 'vagrant-berkshelf (1.2.0)'!

>>vagrant -v
Failed to load the "vagrant-berkshelf" plugin. View logs for more details.

Vagrant version 1.2.2
```

And I try to vagant up with "config.berkshelf.enabled = true" in my VagrantFile, it turns out can not load section of berkshelf

``` bash
>>>vagrant up
Failed to load the "vagrant-berkshelf" plugin. View logs for more details.

Bringing machine 'default' up with 'virtualbox' provider...
There are errors in the configuration of this machine. Please fix
the following errors and try again:

Vagrant:
* Unknown configuration section 'berkshelf'.
```

**Comment by jrstarke** [comment]
I was having a similar problem. Apparently the version of Vagrant included in Ubuntu 14.04 is 1.4.x, and the vagrant-berkshelf plugin requires 1.5.

**Comment by imechemi** [comment]
@tmatilai Thanks, it worked :+1:
